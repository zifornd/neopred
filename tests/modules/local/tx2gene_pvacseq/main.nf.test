nextflow_process {

    name "Test Process TX2GENE_PVACSEQ"
    script "modules/local/tx2gene_pvacseq/main.nf"
    process "TX2GENE_PVACSEQ"

    test("Should run without failures") {

        when {
            process {
                """
                // define inputs of the process here. Example:

		 input[0] = Channel.fromPath("$projectDir/tests/test_result/tx2gene/quants/SRR8281226")
                 input[1] = "salmon"
                 input[2] = file("$projectDir/tests/test_result/tx2gene/gencode.v45.primary_assembly_chr6.annotation.gtf")


                """
            }
        }

        then {
            assert process.success
	    assert process.out.tsv
            assert snapshot(process.out)//.match()
        }

    }

}
