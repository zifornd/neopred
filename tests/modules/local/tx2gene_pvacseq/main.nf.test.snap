{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "tx2gene.tsv:md5,9a0ba31fe431016ee4fb130280ae59a8"
                ],
                "1": [
                    "versions.yml:md5,c95bd38cb268a65c891c4abcb8d08d36"
                ],
                "tsv": [
                    "tx2gene.tsv:md5,9a0ba31fe431016ee4fb130280ae59a8"
                ],
                "versions": [
                    "versions.yml:md5,c95bd38cb268a65c891c4abcb8d08d36"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.1"
        },
        "timestamp": "2025-01-02T09:14:15.231104"
    }
}